exporter/clickhouseexporter/exporter_logs.go

```
@@ -234,7 +311,11 @@ func newClickHouseConn(cfg *Config) (*sql.DB, error) {
 
 	// can return a "bad" connection if misconfigured, we won't know
 	// until a Ping, Exec, etc.. is done
-	return clickhouse.OpenDB(opts), nil
+	conn, err := clickhouse.Open(opts)
+	if err != nil {
+		log.Fatal(err)
```

why fatal? why not return the error?